### ISSUE TITLE
6c - Create Debugging Subsections

### ISSUE THREAD

**Issue by cpholguera** [OWASP/owasp-mstg#1371]
add "##### Debugging Release Apps" subsection to "#### Debugging" before "##### Debugging with lldb"

```
##### Debugging Release Apps
```

Maybe extract common steps from the lldb section that might be applicable for other debuggers like gdb.

Also consider adding more Debugging subsections to "#### Debugging" after "##### Debugging with lldb"

```
##### Debugging with gdb
##### Debugging with an IDE
##### Debugging Native Code
```

Check chapter 5c to keep it consistent.

**Comment by su-vikas** [comment]
I will take this one

**Comment by cpholguera** [comment]
Assigned, thanks ;)

**Comment by su-vikas** [comment]
We dont really need section on  "##### Debugging Native Code" as everything is native in iOS.

**Comment by commjoen** [comment]
ah well: the native code is about using c/c++ (so not using swift/objective-c) as this behaves a bit differently than the objective-C/Swift related materials and might require different tooling as well.

**Comment by su-vikas** [comment]
I get your point, but in terms of using lldb or any other debugger, there is not much difference. More or less things look the same, apart from lets say printing Obj-C class data. 

But if still needed for the sake of completeness, sure we can have section on it too. 
Lets discuss this point again after my current PR is reviewed? It is very much possible I am not aware of some technique, which I am happy to learn about :)

**Comment by cpholguera** [comment]
I'll take a look at the current content and we can discuss afterwards ;)

**Comment by su-vikas** [comment]
@commjoen this one can be closed too?

**Comment by sushi2k** [comment]
Thanks @su-vikas!

Was closed as covered in PR #1552
